cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1062,6 +1071,8 @@ pub(super) mod test {
             # manifest.toml(5) for a list of available options.
             [options]
             systems = []
+            # Uncomment to disable CUDA detection.
+            # cuda-detection = false
```

Comment:
I toyed with "CUDA detection is enabled by default" and whether the commented value should indicate the default or the alternative before ending up on this. Suggestions welcome.